### DISCUSSION TITLE
which content can be used in a search by "m:your-search-term"?

### DISCUSSION THREAD

**Discussion by aisbergde** [immich-app/immich#6215] [Q&A]
from here: https://immich.app/docs/features/search

> Metadata search (prefixed with m:) can search specifically by text without the use of a model.

A user first needs to know which "m:your-search-term" exists / can be used, before using them.

I tried to understand, which content in the database tables could be the data source. And it looks like only the table exif could be a possible source. If this is the case, then which columns in this table are used for the metadata search?

Is it possible to search only by one search term, or is it possible to combine them?

**Comment by aviv926** [comment]
As far as I know, until recently the option allowed to search also within tags created by the object recognition model.
But it was [removed](https://github.com/immich-app/immich/pull/5903) because it is no longer needed due to the use of the CLIP model.
Actually right now this option [doesn't work](https://github.com/immich-app/immich/issues/5982#issuecomment-1869694578) due to the removal of typesense and it will work again in the future.

I assume there will be an expansion on this in the documentation when more filters are added to Immich.

**Reply by aisbergde** [reply]
This is also fine to know. We can wait. I think, there is a rebuild of the search in progress. I have seen, that I can search for some content from some columns from the exif table. But maybe, it is worth, to wait a bit, until the search evolution gets more stable.
